### ISSUE TITLE
Problem reading logbook when ID is not listed in the first column

### ISSUE THREAD

**Issue by cr-xu** [paulscherrerinstitute/py_elog#7]
Hi,

I was trying to read some logbook entries and get this error with the method get_message_ids()

```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File ".../elog-1.3.1-py3.7.egg/elog/logbook.py", line 338, in get_message_ids
  File ".../elog-1.3.1-py3.7.egg/elog/logbook.py", line 338, in <listcomp> 
ValueError: invalid literal for int() with base 10: ''"
```
[](url) https://github.com/paulscherrerinstitute/py_elog/blob/ad66f850fbf3f0ae4ffce4f38f7c8df90b9dda55/elog/logbook.py#L335-L339

The problem is that this method only look for the first column in the elog table and try to convert it into a integer. But the project I'm dealing with has a modified layout of the logbook, and have some other attribute listed in the first column, so this method cannot handle them.

It would be great, (and also more generally applicable) that the method is so implemented, that it first searches for the table header for "ID" and use that column to get the actual message_id.

this problem can be reproduced by changing the demo elog config file as something like
```
...
List display = Date, ID, Author,
Attributes = Author, Type, Category, Subject
...
```

PS: I understand that this can be avoided by just using the standard layout.. but I'm not really authorized to modify the logbook layout...

Thanks,
Chenran

**Comment by simongregorebner** [comment]
Hi Chenran
Thanks for reporting, we'll have a look at it in the next weeks (sorry we're quite busy right now). You are also invited to file a pull-request to fix for this, this would probably speed up things ; )

**Comment by physiker-wfw** [comment]
Hi,

we encountered the same problem for a layout that does not have the ID present! 

In general, a better solution of this issue would be a separate web request that returns the list of all IDs. However, that would be a change of the elog code.

Thanks for consideration, WFW

**Comment by simongregorebner** [comment]
fixed in package version 1.3.2
